Makefile

```diff
@@ -27,6 +27,11 @@ DIST_VER=$(shell git tag | sort -r --version-sort | head -n1)
 -include config.override.mk
 include config.mk
 
+# In case DIST_VER is not assigned, fallback to using the latest semver tag available
```

Comment:
Do we still need line 25?, cause below cause will set to same thing if DIST_VER is not defined.

Reply:
Rebased and pushed changes

Reply:
Amazing, thanks.